metrics/examples/prometheus/alerting-rules.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alerting-rules
+data:
+  alerting-rules.yml: |
+    groups:
+    - name: kafka
+      rules:
+      - alert: UnderReplicatedPartitions
+        expr: kafka_server_replicamanager_underreplicatedpartitions > 0
+        for: 10s
+        labels:
+          severity: warning
+        annotations:
+          summary: 'Kakfa under replicated partitions'
+          description: 'There are {{ $value }} under replicated partitions on {{ $labels.kubernetes_pod_name }}'
+      - alert: NoActiveController
+        expr: sum(kafka_controller_kafkacontroller_activecontrollercount) == 0
```

Comment:
`!= 1`? I know in theory it's impossible for it to be > 1, but stranger things have happened.

Reply:
But it's a "no active controller" alert ... we should change it into a more general "active controller error" ? if we think that it could be > 1 (two potentially controllers at same time) ... do you have a better name?